File: app/src/main/java/com/nelyan/ui/ChangePasswordActivity.java
package com.nelyan.ui;

import androidx.appcompat.app.AppCompatActivity;

import android.app.Dialog;
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.RelativeLayout;

import com.nelyan.R;

public class ChangePasswordActivity extends AppCompatActivity {
Context mContext;
ImageView ivBack;
Button btnSave;
Dialog dialog;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_change_password);
        mContext=this;
        ivBack=findViewById(R.id.ivBack);
        ivBack.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });btnSave=findViewById(R.id.btnSave);
        btnSave.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                showDailog();
            }
        });
    }
    public void showDailog(){

        dialog = new Dialog(mContext);
        dialog.getWindow().setBackgroundDrawableResource(android.R.color.transparent);
        dialog.setContentView(R.layout.alert_change_pass);
        dialog.setCancelable(true);

        Button btnOk;
        btnOk= dialog.findViewById(R.id.btnOk);
        btnOk.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
             //   mContext.startActivity(new Intent(mContext, SettingsActivity.class));
                finish();
                dialog.dismiss();
            }
        });
        dialog.show();
    }
}

File: app/src/main/java/com/nelyan/ui/ChangePasswordActivity.kt
package com.nelyan.ui

import android.app.Dialog
import android.content.Context
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.nelyan.R

class ChangePasswordActivity : AppCompatActivity() {
    var mContext: Context? = null
    var ivBack: ImageView? = null
    var btnSave: Button? = null
    var dialog: Dialog? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_change_password)
        mContext = this
        ivBack = findViewById(R.id.ivBack)
        ivBack!!.setOnClickListener(View.OnClickListener { finish() })
        btnSave = findViewById(R.id.btnSave)
        btnSave!!.setOnClickListener(View.OnClickListener { showDailog() })
    }

    fun showDailog() {
        dialog = Dialog(mContext!!)
        dialog!!.window!!.setBackgroundDrawableResource(android.R.color.transparent)
        dialog!!.setContentView(R.layout.alert_change_pass)
        dialog!!.setCancelable(true)
        val btnOk: Button
        btnOk = dialog!!.findViewById(R.id.btnOk)
        btnOk.setOnClickListener { //   mContext.startActivity(new Intent(mContext, SettingsActivity.class));
            finish()
            dialog!!.dismiss()
        }
        dialog!!.show()
    }
}